localMatchApp/headers/Login.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * \class Login
+ *
+ * \brief Create the connection with the database and the core.
+ *
+ * This class check out the format of the password and check if the
+ * user exists, in the sign-up function.
+ *
+ *
+ */
+
+#ifndef LOGIN_H
+#define LOGIN_H
+
+#include <string>
+class Login {
+ public:
+  /// Create a class type enum to communicate the state of the methods "signUp" and "entry".
+  enum class ConnectionMessage {
+    notEqualPasswords,
+    goodEntry,
+    doesntExists,
+    invalidUser,
+    wrongPassword
+  };
+  /// The constructor connects with the database.
+  Login ( );
+  /// The destructor disconnects the database.
+  ~Login ( );
+  Login ( const Login& ) = default;
+  Login& operator= ( const Login& ) = default;
+
+  /** This method checks the format of the password when a user wants to sign-up in the
+   * application, the correct format of a password has minimum a digit, a capital letter,
+   * and a small letter.*/
+  bool hasCorrectFormat (const std::string &i_password );
+
+  /** This method is created for add a new user in the application database*/
```

Comment:
Done.

Reply:
"This method is created for..." is irrelevant

Reply:
What @wfallascr is trying to say is that you don't have to write in the description that "this is a method" since it is irrelevant or explicit in the code.